.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_MONGODB_READ_TOKEN=D5Kk0zuirvDkz9XpXBt71rN2sRSiU4CeX5CLKJIef7PSWgIKifGZ8IG03fCdxYef
```

Comment:
Please remove this file, the .env file should be in the /app folder and not committed

Reply:
Ok